test/models/lookup_key_test.rb

```diff
@@ -420,4 +420,24 @@ def setup
       assert_valid lookup_key
     end
   end
+
+  test "can update lookup key with default_value without audit error" do
+    location_list = [taxonomies(:location1), taxonomies(:location2)]
+    puppetclass1 = FactoryBot.create(:puppetclass, :with_auditing)
+    environments = FactoryBot.create_list(:environment, 3)
+    environments.each do |e|
+      e.locations = location_list
+      e.puppetclasses = [puppetclass1]
+      e.save!
+    end
```

Comment:
I think a simpler minimal case can be done with something like:
```suggestion
    location = FactoryBot.create(:location)
    environments = FactoryBot.create_list(:environment, 2, locations: [location])
    puppetclass = FactoryBot.create(:puppetclass, environments: environments)
```